src/System.Management.Automation/FormatAndOutput/common/DisplayDatabase/displayDescriptionData_Complex.cs

```diff
@@ -223,9 +223,9 @@ internal static CustomItemBase Create(FormatToken token)
                     cie.Expression = new DisplayEntry(cpt.expression);
                 }
 
-                if (cpt.control != null)
+                if (cpt.control is ComplexControlBody complexControlBody)
```

Comment:
I wander if it is not ComplexControlBody what is it here and why?

Reply:
In the original issue, it's a `ControlReference`.  I don't know why it happens with the `posh` module, but it seems that this type is a reference to a `ComplexControlBody` that's resolved [here](https://github.com/PowerShell/PowerShell/blob/5eacbadb66337eabd75b77e4b480f1087cba9445/src/System.Management.Automation/FormatAndOutput/common/FormatViewGenerator_Complex.cs#L110) so maybe it should do the same thing here

Reply:
The problem is within this abstract class, it doesn't have the necessary info to resolve a `ControlReference`, so nothing can be done here